librz/bin/bobj.c

```diff
@@ -869,14 +869,6 @@ RZ_API RZ_BORROW RzBinString *rz_bin_object_get_string_at(RZ_NONNULL RzBinObject
 	return ht_up_find(obj->strings->phys, address, NULL);
 }
 
-/**
- * \brief Return true if the binary object \p obj is big endian.
- */
-RZ_API bool rz_bin_object_is_big_endian(RzBinObject *obj) {
```

Comment:
I think this can be kept even if it's not currently used... I imagine it should be used when you want to show information of the binary (probably there are places that access info->big_endian directly instead of using this)

Reply:
added back